catalog/nsili/catalog-nsili-sourcestoquery-ui/pom.xml

```diff
@@ -98,7 +98,7 @@
         <dependency>
             <groupId>org.webjars.bower</groupId>
             <artifactId>lodash</artifactId>
-            <version>2.4.1</version>
+            <version>3.7.0</version>
```

Comment:
❗️ According to https://nvd.nist.gov/vuln/detail/CVE-2018-16487, versions `4.17.11+` are not affected so 3.7.0 still has the vulnerability.